src/net/sf/memoranda/ui/DailyItemsPanel.java

```diff
@@ -199,11 +199,11 @@ public void actionPerformed(ActionEvent e) {
         statusPanel.add(indicatorsPanel, BorderLayout.EAST);
 
         mainPanel.add(editorsPanel, BorderLayout.CENTER);
-        
+        PSPControlPanel PSPPanel = new PSPControlPanel();
```

Comment:
According to our new Coding Standards, this variable name needs to be lower camel case (`pspPanel`)